admiral/pkg/clusters/serviceentry_test.go

```diff
@@ -469,3 +469,72 @@ func TestCreateServiceEntry(t *testing.T) {
 	}
 
 }
+
+func TestCreateIngressOnlyVirtualService(t *testing.T) {
+
```

Comment:
This test could be cleaned up a bit using the iterative test format: https://gist.github.com/josephpeacock/6f5a73c3f133029d59ab736c83c1b087 - let's you avoid duplicating the validation logic and makes it easier to add cases in the future if needed

Reply:
Good suggestion. Updated.